extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js

```diff
@@ -40,6 +120,20 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',[
 		return geo.isProperGeo(instantGlobals.wgAdDriverIndexExchangeBidderCountries);
 	}
 
+	function getSlots(skin, pageType) {
```

Comment:
how about doing something similar like in openxbidder?
```javascript
	function setupHomePageSlots() {
		var slots = getSlots();

		Object.keys(slots).forEach(function (slotName) {
			if (slotName.indexOf('TOP') > -1) {
				addSlot('HOME_' + slotName, slots[slotName]);
				removeSlot(slotName);
			}
		});

		addSlot('PREFOOTER_MIDDLE_BOXAD', {sizes: ['300x250']});
	}
```

maybe this logic can be extracted an shared among bidders? We would need to add `addSlot` and `removeSlot`